adapters/frvradn/frvradn.go

```diff
@@ -98,7 +98,7 @@ func (a *adapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.R
 	var errs []error
 	for _, seatBid := range response.SeatBid {
 		for i, bid := range seatBid.Bid {
-			bidType, err := getBidMediaType(bid)
+			bidType, err := getBidMediaType(&bid)
```

Comment:
you are still using copy of seatBid.Bid
try using bidType, err := getBidMediaType(seatBid.Bid[i])